lib/ansible/modules/yum_repository.py

```diff
@@ -548,23 +589,23 @@ def main():
     # Module settings
     argument_spec = dict(
         bandwidth=dict(),
-        baseurl=dict(type='list'),
+        baseurl=dict(type='list', elements='str'),
         cost=dict(),
         deltarpm_metadata_percentage=dict(),
         deltarpm_percentage=dict(),
         description=dict(),
-        enabled=dict(type='bool'),
-        enablegroups=dict(type='bool'),
-        exclude=dict(type='list'),
+        enabled=dict(type='bool', default=True),
+        enablegroups=dict(type='bool', default=True),
```

Comment:
The defaults here also change behavior IMO. Even if these are the default values used by yum, this could lead to repository files being changed.

Reply:
So maybe this has to remove from docs too instead of adding it to argument_spec?

Reply:
Yes. This should probably be mentioned in the description, that yum's default value is `true`, but not in the `default:` value.

Reply:
Could you please check the description format and see if it's OK or not?

Reply:
I'd use `C(true)` for the value; besides that, yes.

Reply:
Could you please explain more?

Reply:
> I'd use `C(true)` for the value; besides that, yes.

Fixed in https://github.com/ansible/ansible/pull/70319/commits/66ec490a8acf5441329564f6c6343c5b14c68f50.